src/channels/vonage/renderers/choices.ts

```diff
@@ -3,11 +3,15 @@ import { ChoicesRenderer } from '../../base/renderers/choices'
 import { VonageContext } from '../context'
 
 export class VonageChoicesRenderer extends ChoicesRenderer {
-  renderChoice(context: VonageContext, payload: ChoiceContent): void {
+  renderChoice(context: VonageContext, payload: ChoiceContent) {
     const message = context.messages[0]
 
     message.content.text = `${message.content.text}\n\n${payload.choices
       .map(({ title }, idx) => `*(${idx + 1})* ${title}`)
       .join('\n')}`
+
+    if (context.identity && context.sender) {
```

Comment:
Is this check necessary? It'll leave it in but it seems overly cautious. Is there any situation where this isn't set?